controller/src/main.cpp

```diff
@@ -72,13 +72,19 @@ limitations under the License.
 //
 // "Sends" data to the "GUI" via a simple serial protocol.  This can be parsed
 // and graphed by e.g. the Arduino IDE (tools -> serial plotter).
+static DebugUInt32 gui_mode("gui_mode", "Mode setting for GUI free testing");
+static DebugUInt32 gui_bpm("gui_bpm", "Breaths/min for GUI free testing");
```

Comment:
Perhaps use "rr" as the more common term?

Reply:
rr?  Not sure what you mean here